frontend/packages/ceph-storage-plugin/src/components/block-pool/body.tsx

```diff
@@ -110,6 +118,12 @@ export const BlockPoolBody = (props: BlockPoolBodyPros) => {
 
   // Check storage cluster is in ready state
   const isClusterReady: boolean = cephCluster?.status?.phase === POOL_STATE.READY;
+  React.useMemo(() => {
```

Comment:
```suggestion
  React.useEffect() => {
```
useMemo should return a value.

Reply:
ack